src/Symfony/Component/Runtime/composer.json

```diff
@@ -20,9 +20,10 @@
         "composer-plugin-api": "^1.0|^2.0"
     },
     "require-dev": {
-        "composer/composer": "^1.0.2|^2.0",
+        "composer/composer": "^2.6",
```

Comment:
[Version 2.6](https://packagist.org/packages/composer/composer#dev-main) brings support of Symfony 7.0